app/forms/validators.py

```diff
@@ -424,3 +421,26 @@ def __call__(self, answer_values, is_mandatory):
                 self.messages["MANDATORY_QUESTION"], self.question_title
             )
             raise validators.ValidationError(message)
+
+
+def sanitise_mobile_number(data):
+    for char in [" ", ".", ",", "\t", "-", "{", "}", "[", "]", "(", ")", "/"]:
+        data = data.replace(char, "")
+
+    return data.lstrip("+044").lstrip("044").lstrip("0")
```

Comment:
`lstriip` Removes all leading character specified by the argument.
```py
data.lstrip("+044")
```
However, I am not sure `lstrip` is what we want. It does make it more flexible but it strips any number of the specified characters, for example, it would accept numbers like `+440000044x xxx xxx xxx` as it would interpret this as `xxxxxxxxxx`.